src/commands/package.js

```diff
@@ -37,8 +39,8 @@ exports.builder = yargs => {
 };
 
 exports.handler = function(argv = {}) {
-	const { token, search, limit, regex, json } = argv;
-	const { errors, repositories } = getRepositories(limit);
+	const { token, search, limit, regex, json, repoList } = argv;
+	const { errors, repositories } = getRepositories({ limit, repoList });
```

Comment:
Add a `const repos = repo;` somewhere to improve this.

```suggestion
	const { errors, repositories } = getRepositories({ limit, repo });
```

Reply:
added in const repoList = repo, so not needed.